### PR TITLE
[Fix] Tune FF instead of D via inflight adjustments if PIFF controller is active

### PR DESCRIPTION
**Pull request by Scavanger**
Fix for #5990 

PR for configurator: https://github.com/iNavFlight/inav-configurator/pull/1053

The table in the documentation was also corrected on that occasion.

### CODE REVIEW

File: docs/Inflight Adjustments.md [hunk 1]
```diff
@@ -113,45 +113,45 @@ this reason ensure that you define enough ranges to cover the range channel's us
...
+| 40    | POS_Z_D |
+| 41    | HEADING_P |
+| 42    | VEL_XY_P |
+| 43    | VEL_XY_I |
+| 44    | VEL_XY_D |
 | 45    | VEL_Z_P |
 | 46    | VEL_Z_I |
 | 47    | VEL_Z_D |
 | 48    | FW_MIN_THROTTLE_DOWN_PITCH_ANGLE |
-| 49    | PROFILE | Switch between 3 rate profiles using a 3 position switch. |
```
Comment (DzikuVx) [inline docs/Inflight Adjustments.md:154]: @Scavanger was this intended?

**Review by DzikuVx — CHANGES_REQUESTED** [review]
@Scavanger could you redo the PR in a way that only a new functions are added and existing are not removed?

File: docs/Inflight Adjustments.md (hunk 1 above)
Comment (Scavanger) [reply docs/Inflight Adjustments.md:154]: It's just a fix in the documentation, rate profile is currently inactive in the code.

**Comment by Scavanger** [comment]
@DzikuVx I didn't remove functions, just  a fix in the documentation.

**Comment by DzikuVx** [comment]
OK then, let's have it merged then. Thanks

**Pull request merged**